### PR TITLE
[MIGraphX EP Support]Remove default noopt for Migraphx EP in Benchmark.py

### PR DESCRIPTION
**Pull request by TedThemistokleous**
…ripts (#58)

### Description

Removes the heavy handed no opt for all MIGraphX using the benchmark.py scripts

### Motivation and Context

Finding this hurts performance if we remove all optimizations. Let the fine tuning occur at the script level instead of a blanket NoOPT being selected

### CODE REVIEW

**Comment by TedThemistokleous** [comment]
ping @Cloudhan @tianleiwu  doesn't effect the ep/code any but more support using your benchmark scripts.

File: onnxruntime/python/tools/transformers/benchmark.py [hunk 1]
```diff
@@ -135,15 +135,6 @@ def run_onnxruntime(
             )
             return results
 
-    if provider == "migraphx":
-        optimizer_info = OptimizerInfo.NOOPT
-        warm_up_repeat = 5
-        if "MIGraphXExecutionProvider" not in onnxruntime.get_available_providers():
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/benchmark.py:141]: This warning might still be helpful?

Comment (TedThemistokleous) [reply onnxruntime/python/tools/transformers/benchmark.py:141]: Its just reverting things from the previous state

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**